jobs/config.json

```diff
@@ -9940,7 +9940,11 @@
   },
   "maintenance-boskos-config-upload": {
     "args": [
-      "cd boskos && make update-config"
+      "make",
+      "--",
```

Comment:
you don't want the `--` before `-C`, as that will make `make` not treat `-C` as a flag to `make`.

Reply:
oh, maybe you need it so execute.py doesn't interpret the `-C`?

this is confusing.

Reply:
I would expect it to work with or without the --